tests/integration/mwss/test_mwss_submission_data.py

```diff
@@ -0,0 +1,326 @@
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestMwssSubmissionData(IntegrationTestCase):
+
+    def test_mwss_data_matches_expected_1_0005(self):
+        self.complete_survey('1', '0005')
+
+        # And the JSON response contains the data I submitted
+        actual_downstream_data = self.dumpSubmission()
+
+        self.assertDictEqual(actual_downstream_data, self.get_expected_submission_data(actual_downstream_data))
+
+    @staticmethod
+    def get_expected_submission_data(actual_downstream_data):
```

Comment:
Might be cleaner to pass the tx_id and submitted_at directly rather than the the whole actual submission. Up to you.